modules/ravestate/debounce.py

```diff
@@ -0,0 +1,19 @@
+from threading import Timer
+
+
+def debounce(wait):
```

Comment:
We can implement debounce with a state and a signal, this decorator is not gonna work with ravestate